scripts/lib/CIME/get_timing.py

```diff
@@ -459,11 +459,10 @@ def _getTiming(self, inst=0):
             xmaxr = adays*86400.0/(xmax*365.0)
         tmaxr = 0
         if tmax > 0:
-            print ("HERE adays {} tmax {}".format(adays, tmax))
             tmaxr = adays*86400.0/(tmax*365.0)
 
         self.write("\n")
-        self.write("  total pes active           : {} \n".format(totalpes*maxthrds*smt_factor ))
+        self.write("  total pes active           : {} \n".format(totalpes*smt_factor ))
         self.write("  mpi tasks per node               : {} \n".format(max_mpitasks_per_node))
         self.write("  pe count for cost estimate : {} \n".format(pecost))
```

Comment:
Since you're here anyways, could you consider updating this?
1. Indentation of `:` for "mpi tasks per node" line
2. It's hard to estimate the cost of a job: e.g. "charge factor" at NERSC Cori. I think it'd be useful to simplify and write
```
  total nodes         : N (num_nodes)
  mpi tasks per node  : M (MAX_MPITASKS_PER_NODE)
  threads per mpi task: T (max_threads)
  pes per node        : MxT
  total pes active    : MxTxN
```
i.e. remove `smt_factor` from all PE calculations. It's hard to account for SMT/hyper-threading correctly without introducing additional XML variable for actual number of cores per node: e.g. OLCF Summit node with 42 cores per node capable of SMT[1,2,4].

Reply:
I have addressed the indentation.  The smt_factor change is more extensive and not something I feel can be handled as a minor addition.  I would be happy to review a new PR if you would like to submit one.

Reply:
One-line change:
```
diff --git a/scripts/lib/CIME/get_timing.py b/scripts/lib/CIME/get_timing.py
index 68c2e7769..6e197ead4 100644
--- a/scripts/lib/CIME/get_timing.py
+++ b/scripts/lib/CIME/get_timing.py
@@ -405,7 +405,7 @@ class _TimingParser:
                 comp_label = m.comp + inst_label
             else:
                 comp_label = m.comp
-            self.write("  {} = {:<8s}   {:<6d}      {:<6d}   {:<6d} x {:<6d}  {:<6d} ({:<6d}) \n".format(m.name.lower(), comp_label, (m.ntasks*m.nthrds *smt_factor), m.rootpe, m.ntasks, m.nthrds, m.ninst, m.pstrid))
+            self.write("  {} = {:<8s}   {:<6d}      {:<6d}   {:<6d} x {:<6d}  {:<6d} ({:<6d}) \n".format(m.name.lower(), comp_label, (m.ntasks*m.nthrds), m.rootpe, m.ntasks, m.nthrds, m.ninst, m.pstrid))
             if m.nthrds > maxthrds:
                 maxthrds = m.nthrds
         if self._driver == 'nuopc':
```